server/integration/endpoint/src/main/java/org/infinispan/server/endpoint/EndpointLogger.java

```diff
@@ -184,5 +185,9 @@
    @LogMessage(level = INFO)
    @Message(id = 10032, value = "No Security realm declared for endpoint '%s'. Using default encryption realm.")
    void noSSLContextForSni(String endpoint);
+
+   @LogMessage(level = INFO)
+   @Message(id = 10033, value = "Starting server %s without transport (use multi tenant router instead)")
```

Comment:
This sounds a bit like a warning to me... Could we make it an error and log it only when the multi-tenant router is not configured?